docs/utils/testing.rst

```diff
@@ -0,0 +1,50 @@
+.. _pyvo-testing:
+
+******************************************
+Helpers for Testing (`pyvo.utils.testing`)
```

Comment:
bikeshed  at this point - not sure how to do this best, but I feel these should be private functions (aka for consumptions only within pyvo), yet I agree that having API docs for them would be nice rather than hiding away.

maybe just an extra sentence that say that?